scripts/daam_script.py

```diff
@@ -124,27 +138,33 @@ def run(self,
         # print(f"hijack_comments={prompt_analyzer.hijack_comments}, used_custom_terms={prompt_analyzer.used_custom_terms}")
         # print(f"fixes={prompt_analyzer.fixes}")
         
-        if any(item[0] in attentions for item in self.prompt_analyzer.used_custom_terms):
+        if any(item[0] in self.attentions for item in self.prompt_analyzer.used_custom_terms):
             print("Embedding heatmap cannot be shown.")
             
         global before_image_saved_handler
         before_image_saved_handler = lambda params : self.before_image_saved(params)
-                
-        with torch.no_grad():
-            with trace(p.sd_model, p.height, p.width, context_size) as tr:
-                self.tracer = tr
-                               
-                processed = process_images(p)
```

Comment:
Since `process_images()` is no longer executed by the extension, we must hook and unhook `trace` manually.

It is hooked here, and later unhooked in the `postprocess` function